graphql-ballerina/build.gradle

```diff
@@ -124,159 +89,14 @@ task updateTomlFiles {
     }
 }
 
-task revertTomlFiles {
+task revertTomlFile {
     doLast {
         ballerinaTomlFile.text = originalBallerinaToml
         dependenciesTomlFile.text = originalDependenciesToml
         compilerPluginTomlFile.text = originalCompilerPluginToml
     }
 }
```

Comment:
Is this function still needed in the plugin?

Reply:
Plugin does not has a revertTomlFile task. It only declares build finalized by revertTomlFile

Reply:
Ack